Sources/MQTTNIO/MQTTClient.swift

```diff
@@ -11,6 +11,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Atomics
```

Comment:
I think you can remove `NIOConcurrencyHelpers` now right?

Reply:
I'm using `Lock` so I can't. There are a couple of other files though where I can remove it